sdk/tests/deqp/functional/gles3/es3fMultisampleTests.js

```diff
@@ -617,13 +617,17 @@ goog.scope(function() {
 
         /** @type {number} */ var requiredNumDistinctColors = this.m_numSamples + 1;
 
+        // If the number of samples is high (64 or more), we need to lower the threshold for detecting unique colors, otherwise two expected unique colors would be treated as the same color.
+        var threshold = Math.min(3, Math.floor(255 / this.m_numSamples) - 1);
```

Comment:
Sorry, after merge, just realized this can end up negative.  @Oletus Can you address it in a follow up CL?